### ISSUE TITLE
Upgrade from 0.6.1 to 0.6.5 breaks icons like "mdi-microphone"

### ISSUE THREAD

**Issue by XStarlink** [nuxt-modules/icon#119]
Hello, I've just updated nuxt-icon to 0.6.5 and the icons in my app that I had in this form no longer display correctly, I just have text instead of icons.

**Example of icons that no longer work**
mdi-microphone
mdi-plus
mdi-close
etc...

**Comment by Atinux** [comment]
Hi @XStarlink 

It's hard to follow all of the different format.

Could you try by prefixing with `i-` ? or using `mdi:plus` format?

**Comment by XStarlink** [comment]
@Atinux I finally prefixed all my icons with i- to unify the use of icons in my code!

Thanks for all your work on nuxt, it's the best framework ever!
